base/applications/rapps/gui.cpp

```diff
@@ -1469,6 +1469,19 @@ class CMainWindow :
         }
     }
 
+    VOID ShowAboutDlg()
+    {
+        ATL::CStringW szApp;
+        ATL::CStringW szAuthors;
+        HICON hIcon;
+
+        szApp.LoadStringW(IDS_APPTITLE);
+        szAuthors.LoadStringW(IDS_APP_AUTHORS);
+        hIcon = (HICON) LoadIconW(hInst, MAKEINTRESOURCEW(IDI_MAIN));
+        ShellAboutW(m_hWnd, szApp, szAuthors, hIcon);
+        DeleteObject(hIcon);
```

Comment:
Icons have their own function to destroy the icon and free up the handle from the memory space, `DestroyIcon()`. Use that instead.

Reply:
ok